docs/en/ingest-management/elastic-agent/example-kubernetes-fleet-managed-agent-helm.asciidoc

```diff
@@ -0,0 +1,152 @@
+[[example-kubernetes-fleet-managed-agent-helm]]
+= Example: Install {fleet}-managed {agent} on Kubernetes using Helm
+
+preview::[]
+
+This example demonstrates how to install {fleet}-managed {agent} on a Kubernetes system using a Helm chart, gather Kubernetes metrics and send them to an {es} cluster in {ecloud}, and then view visualizations of those metrics in {kib}.
+
+For an overview of the {agent} Helm chart and its benefits, refer to <<install-on-kubernetes-using-helm>>.
+
+This guide will take you through the following steps:
+
+* <<agent-fleet-managed-helm-example-install-agent>>
+* <<agent-fleet-managed-helm-example-install-integration>>
+* <<agent-fleet-managed-helm-example-tidy-up>>
+
+
+[discrete]
+[[agent-fleet-managed-helm-example-prereqs]]
+=== Prerequisites
+
+To get started, you need:
+
+* A local install of the link:https://helm.sh/[Helm] Kubernetes package manager.
+* An link:{ess-trial}[{ecloud}] hosted {es} cluster on version 8.16 or higher.
```

Comment:
Global: References to `8.16`  is peppered throughout.  This PR won't be backported beyond 8.16 so a lot of those instances--but probably not all--are unnecessary. 
Restated: If a user is running 8.15 and using the 8.15 docs, for example, they wouldn't be seeing this new content anyway.

Reply:
Please see my comment below.